services/core-web/src/components/mine/Compliance/ComplianceOrdersTable.js

```diff
@@ -67,70 +55,33 @@ const columns = [
     dataIndex: "overdue",
     render: (text, record) => (
       <div title="Overdue">
-        {record.overdue && record.due_date !== null ? (
+        {text && record.due_date !== null ? (
           <img className="padding-sm" src={RED_CLOCK} alt="Overdue Report" />
         ) : (
           ""
         )}
       </div>
     ),
   },
-  {
-    title: "Order No.",
-    key: "order_no",
-    dataIndex: "order_no",
-    render: (text, record) => <div title="Order No.">{record.order_no || "-"}</div>,
-    sorter: (a, b) => (a.order_no > b.order_no ? -1 : 1),
-  },
+  renderTextColumn("order_no", "Order No.", true, "-"),
   {
     title: "Violation",
     key: "violation",
     dataIndex: "violation",
-    render: (text, record) => <div title="Violation">{record.violation || "-"}</div>,
+    render: (text) => <div title="Violation">{text || "-"}</div>,
     sorter: (a, b) => compareCodes(a.violation, b.violation),
   },
-  {
-    title: "Report No.",
-    key: "report_no",
-    dataIndex: "report_no",
-    render: (text, record) => <div title="Report No.">{record.report_no || "-"}</div>,
-    sorter: (a, b) => (a.report_no > b.report_no ? -1 : 1),
-  },
-  {
-    title: "Inspector",
-    key: "inspector",
-    dataIndex: "inspector",
-    render: (text, record) => <div title="Inspector">{record.inspector || "-"}</div>,
-    sorter: (a, b) => (a.inspector > b.inspector ? -1 : 1),
-  },
-  {
-    title: "Order Status",
-    key: "order_status",
-    dataIndex: "order_status",
-    render: (text, record) => <div title="Order Status">{record.order_status || "-"}</div>,
-    sorter: (a, b) => (a.order_status > b.order_status ? -1 : 1),
-  },
+  renderTextColumn("report_no", "Report No.", true, "-"),
```

Comment:
Big fan of this `renderTextColumn` in general.  These columns are so bulky, this will make things much cleaner.